LDAR_Sim/src/default_parameters/virtual_world_default.yml

```diff
@@ -22,6 +22,7 @@ repairs:
     # or the name of a column in the repair delays file to sample from
 emissions:
   emissions_file: "_placeholder_str_" # File containing sample emissions rates or emissions distributions.
+  Pre-Simulation Emissions: True # True/False of whether to simulate emissions before the start date of the simulation
```

Comment:
nit: why is this the only parameter that's both capitalized and with spaces not dashes

Reply:
Resolving this as it is the format we eventually want to pursue